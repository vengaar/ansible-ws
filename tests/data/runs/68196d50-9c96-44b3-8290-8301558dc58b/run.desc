{"playbook": "/home/vengaar/wapi/test/playbooks/groupvars.yml", "cmdline": "ansible-playbook /home/vengaar/wapi/test/playbooks/groupvars.yml -v -i ~/wapi/test/inventories/hosts --extra-vars '{\"country_selected\":\"fr\",\"countries_selected\":[\"de\",\"it\"]}'", "options": "-v -i ~/wapi/test/inventories/hosts", "country_selected": "fr", "countries_selected": "de,it"}