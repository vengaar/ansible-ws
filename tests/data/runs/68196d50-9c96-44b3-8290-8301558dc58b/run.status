{"pid": 19707, "runid": "68196d50-9c96-44b3-8290-8301558dc58b", "status": "finished", "state": "succeeded", "begin": 1551460770.1356053, "end": 1551460772.3612068, "return_code": 0, "description": {"playbook": "/home/vengaar/wapi/test/playbooks/groupvars.yml", "cmdline": "ansible-playbook /home/vengaar/wapi/test/playbooks/groupvars.yml -v -i ~/wapi/test/inventories/hosts --extra-vars '{\"country_selected\":\"fr\",\"countries_selected\":[\"de\",\"it\"]}'", "options": "-v -i ~/wapi/test/inventories/hosts", "country_selected": "fr", "countries_selected": "de,it"}}