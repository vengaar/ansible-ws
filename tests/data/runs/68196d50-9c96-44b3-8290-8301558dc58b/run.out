[0;34mUsing /home/vengaar/.ansible.cfg as config file[0m
[0;34m/home/vengaar/wapi/test/inventories/hosts did not meet host_list requirements, check plugin documentation if this is unexpected[0m
[0;34m/home/vengaar/wapi/test/inventories/hosts did not meet script requirements, check plugin documentation if this is unexpected[0m

PLAY [localhost] ***************************************************************

TASK [Display country_selected] ************************************************
[0;32mok: [localhost] => {[0m
[0;32m    "country_selected": "fr"[0m
[0;32m}[0m

TASK [Display countries_selected] **********************************************
[0;32mok: [localhost] => {[0m
[0;32m    "countries_selected": [[0m
[0;32m        "de",[0m
[0;32m        "it"[0m
[0;32m    ][0m
[0;32m}[0m

TASK [Check extra vars type] ***************************************************
[0;32mok: [localhost] => {[0m
[0;32m    "changed": false,[0m
[0;32m    "msg": "All assertions passed"[0m
[0;32m}[0m

PLAY RECAP *********************************************************************
[0;32mlocalhost[0m                  : [0;32mok=3   [0m changed=0    unreachable=0    failed=0   

